redaxo/src/addons/metainfo/install.php

```diff
@@ -30,11 +30,14 @@
 $managers = [
     'article' => new rex_metainfo_table_manager(rex::getTable('article')),
     'media' => new rex_metainfo_table_manager(rex::getTable('media')),
+    'clang' => new rex_metainfo_table_manager(rex::getTable('clang')),
 ];
 for ($i = 0; $i < $sql->getRows(); ++$i) {
     $column = $sql->getValue('name');
```

Comment:
Wir brauchen hier die Table ohne Prefix, und wir sind hier in der install.php, da existieren die Funktionen nicht. Das ließe sich natürlich beheben, würde es dann aber wohl so lassen.